shared/chat/conversation/messages/cards/team-journey/index.tsx

```diff
@@ -19,6 +19,8 @@ type Props = {
   textComponent: React.ReactNode
 }
 
+const buttonSpace = 6
```

Comment:
Would move this right above `const styles` unless it's used elsewhere that I'm not seeing.